tests/test_distance.py

```diff
@@ -190,3 +192,43 @@ def test_closest_contact_nan_pos():
     xyz = xyz[:-1]
     traj = md.Trajectory(xyz=xyz, topology=None)
     _verify_closest_contact(traj)
+
+
+def test_distance_t_inputs():
+    incorrect_pairs = np.array((0, ptraj.n_atoms+1))
+    with pytest.raises(ValueError):
```

Comment:
Just to make sure the right `ValueError` is thrown
```suggestion
    with pytest.raises(ValueError, match='atom_pairs'):
```